tests/kani/Stubbing/glob_path.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// kani-flags: --harness check_stub --enable-unstable --enable-stubbing
+//! Test that stub can solve glob cycles even when the path expands the cycle.
+
+pub mod mod_a {
+    pub use crate::mod_b::*;
+    pub use crate::*;
+
+    /// This method always fail.
+    pub fn method_a() {
+        mod_a::mod_b::mod_a::mod_b::noop();
+        panic!();
+    }
+}
+
+pub mod mod_b {
+    pub use crate::mod_a::*;
+    pub use crate::*;
+
+    /// This harness replace `method_a` which always fail by `method_b` that should always succeed.
```

Comment:
Took me some time to understand. I'd suggest:
```suggestion
    /// This harness replaces `method_a` (always fails), by `method_b` (always succeeds).
```